src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,99 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  height: 48px;
+  align-items: center;
+  padding-left: 20px;
+  width: 100%;
+  white-space: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+
+  @include for-desktop-up {
+    height: 80px;
+    padding-left: 30px;
+  }
+}
+
 .link {
   display: inline-block;
-  padding: 10px;
+  padding: 0 20px;
   text-decoration: none;
+  height: 23px;
+  border-radius: 100px;
+  font-size: 12px;
```

Comment:
We shouldn't need `font-size` in the CSS anywhere, there's a typography style for this (e.g. `@include type-nav-horizontal`).

You can add breakpoints into the typography styles, you'll see I've done this in my PR.

Reply:
Which PR, @davidgilbertson ?